ExpandRegion.py

```diff
@@ -70,6 +70,7 @@ def run(self, edit, language="", undo=False, debug=True):
       print("Determined language: '{0}'".format(language))
 
     new_regions = []
+    call_sublime_expand_scope = False
```

Comment:
Improve naming. Maybe `expand_region_done`? Or `did_expand_region`?

Reply:
I inverted the name to `is_region_expanded`.